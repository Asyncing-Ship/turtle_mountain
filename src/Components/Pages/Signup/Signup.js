import React, { Component } from 'react';
import './Signup.css';

class Signup extends Component {
  render() {
    return (
      <>
        This is the signup page!
      </>
    )
  }
}

export default Signup;