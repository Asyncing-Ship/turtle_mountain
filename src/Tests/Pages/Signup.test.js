import React from "react";
import { shallow, mount } from "enzyme";
import Signup from "../../Components/Pages/Signup/Signup";
import toJson from "enzyme-to-json";
import { createStore } from "redux";
import rootReducer from "../../Redux/Reducers/reducers";
import { NavLink } from "react-router-dom";
import { Button } from "@chakra-ui/core";
const store = createStore(rootReducer);
it("renders without crashing", () => {
  shallow(<Signup />);
});
it("renders without crashing", () => {
  const mockColor = "David";
  const wrapper = shallow(<Signup color={mockColor} />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
it("renders without crashing", () => {});
