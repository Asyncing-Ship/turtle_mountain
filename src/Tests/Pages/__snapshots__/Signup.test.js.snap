// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Fragment>
  This is the signup page!
</Fragment>
`;
